ajax/libs/videojs-background/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-background",
+  "author": "Matthew Harrison-Jones",
+  "filename": "videojs-background.min.js",
+  "description": "Full background video for video.js",
+  "license": "MIT",
+  "keywords": [
+    "videojs",
+    "html5",
+    "flash",
+    "video",
+    "player",
+    "background"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthojo/videojs-Background.git"
+  },
+  "npmName": "videojs-background",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
+      ]
```

```suggestion
      "basePath": "",
      "files": [
        "lib/**/*"
      ]
```

I would suggest we expand `dist` to the root of the package, but then leave `lib` in its directory.